diff --git a/msvc/config.h b/msvc/config.h
index c44d377..626c733 100644
--- a/msvc/config.h
+++ b/msvc/config.h
@@ -23,7 +23,7 @@
  * match your WinUSB redist directories.
  */
 #ifndef WDK_DIR
-#define WDK_DIR "C:/Program Files (x86)/Windows Kits/8.0"
+#define WDK_DIR "../wdk/Windows Kits/8.0"
 #endif
 
 /* WDK WDF coinstaller version */
@@ -37,12 +37,12 @@
 
 /* embed libusb0 driver files from the following location */
 #ifndef LIBUSB0_DIR
-#define LIBUSB0_DIR "D:/libusb-win32"
+// #define LIBUSB0_DIR "C:\\Program Files\\LibUSB-Win32"
 #endif
 
 /* embed libusbK driver files from the following location */
 #ifndef LIBUSBK_DIR
-#define LIBUSBK_DIR "D:/libusbK/bin"
+// #define LIBUSBK_DIR "D:/libusbK/bin"
 #endif
 
 /* embed user defined driver files from the following location */
